tests/test_logging_config.py

```diff
@@ -95,6 +95,55 @@
 }
 """
 
+SETTINGS_FILE_WASB_HANDLER = """
+REMOTE_BASE_LOG_FOLDER = 'wasb'
```

Comment:
Would it be possible to test this with the _default_ logging config  and the core__remote_logging settings instead?

Reply:
Something like this might help

```python
from six.moves import reload_module

def x():
   import airflow.config_templates.airflow_local_settings
   reload_module(airflow.config_templates.airflow_local_settings)
```

Reply:
> Would it be possible to test this with the _default_ logging config and the core__remote_logging settings instead?

I dug through the code for loading the log configs for remote logging. 
Looks like its as simple as changing the following settings:
1. `core__remote_logging`
2. `core__remote_log_conn_id`
3. `core__remote_base_log_folder`

This is in line with the changes that need to be made in order to [write logs to S3 or GCS.](https://airflow.apache.org/howto/write-logs.html#writing-logs-to-amazon-s3)
The docs describe a longer process for [writing logs to Azure Blob Storage](https://airflow.apache.org/howto/write-logs.html#writing-logs-to-azure-blob-storage). 
The docs also need to be updated. Will update them in a separate [AIRFLOW-XXX] PR.
Updating the test.

Thanks @ashb